source/mainnet/net/references/grpc.rst

```diff
@@ -468,33 +488,34 @@ The node
 
 .. function:: DumpStop() -> Bool
 
-   Stop dumping packages.
+   Stop dumping packages. *Only available on a node built with the
+   network_dump feature.*
 
    :returns: Whether it stopped dumping correctly.
    :rtype: Bool
 
 Networks and peers
 ==================
 
-.. function:: PeerList(includeBootstrappers) -> PeerListResponse
+.. function:: PeerList(include_bootstrappers: Bool) -> PeerListResponse
 
    Get a list of the peers that the node is connected to.
 
-   :param Bool includeBootstrappers: Whether to include the bootstrapper nodes
+   :param Bool include_bootstrappers: Whether to include the bootstrapper nodes
                                      in the response.
    :returns: A list of peers.
    :rtype: |PeerListResponse|_
```

Comment:
Perhaps give more details of the fields of the result message?

Reply:
The fields are described in the protobuf file that is linked.  Is that not sufficient?

Reply:
I suppose so.